src/main/java/teammates/ui/automated/AutomatedServlet.java

```diff
@@ -28,6 +28,10 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) {
     @SuppressWarnings("PMD.AvoidCatchingThrowable") // used as fallback
     public void doPost(HttpServletRequest req, HttpServletResponse resp) {
         try {
+            if (req.getParameterNames().hasMoreElements()) {
```

Comment:
I don't think you need this condition. Without this, an empty request param will result in `{}` being logged which is not wrong at all.

Reply:
All the cron jobs will show `{}` though; not very clean?

Reply:
Which is the case before, no?
Although really this is just a minor nit and you can ignore if you wish.

Reply:
Currently the cron job logs don't show request parameters at all (unless there's an error), which makes sense because there aren't any. Think I will stick to this.